source/adminguide/accounts.rst

```diff
@@ -337,15 +337,15 @@ the user are used.
        #. CloudStack searches for it in LDAP by the configured
           ``ldap.username.attribute``.
 
```

Comment:
@Spaceman1984 The changes in this file, are made in other PR as well: https://github.com/apache/cloudstack-documentation/pull/225, either remove it from one PR (#223 or #225) or make sure conflicts are resolved properly.

Reply:
This will not cause a conflict @sureshanaparti

Reply:
I'll remove. no problem